src/nspawn/nspawn-stub-pid1.c

```diff
@@ -171,18 +171,12 @@ int stub_pid1(sd_id128_t uuid) {
                 /* Would love to use a switch() statement here, but SIGRTMIN is actually a function call, not a
                  * constant… */
 
-                if (si.si_signo == SIGRTMIN+3 ||
-                    si.si_signo == SIGRTMIN+4 ||
-                    si.si_signo == SIGRTMIN+13 ||
-                    si.si_signo == SIGRTMIN+14)
+                if (IN_SET(si.si_signo - SIGRTMIN, 3, 4, 13, 14))
 
                         state = STATE_POWEROFF;
 
                 else if (si.si_signo == SIGINT ||
-                         si.si_signo == SIGRTMIN+5 ||
-                         si.si_signo == SIGRTMIN+6 ||
-                         si.si_signo == SIGRTMIN+15 ||
-                         si.si_signo == SIGRTMIN+16)
+                         IN_SET(si.si_signo - SIGRTMIN, 5, 6, 15, 16))
```

Comment:
Here too. Let's just include all the signal numbers in a single expression.

Reply:
OK. Dropped.